server/utils.go

```diff
@@ -20,13 +21,12 @@ import (
 
 	"github.com/tinode/chat/server/auth"
 	"github.com/tinode/chat/server/store/types"
-	"github.com/ttacon/libphonenumber"
+	"github.com/nyaruka/phonenumbers"
 
 	"golang.org/x/crypto/acme/autocert"
 )
 
 var tagPrefixRegexp = regexp.MustCompile(`^([a-z]\w{0,5}):\S`)
```

Comment:
One more thing: the `^([a-z]\w{0,5}):` isn't quite right:
https://github.com/tinode/chat/blob/master/server/utils.go#L47

A TODO is OK for now though.

Reply:
Changed to checking for any alphanumeric prefixes.